extract_images.py

```
@@ -1,3 +1,96 @@
+        popen = subprocess.Popen(cmd, universal_newlines=True, shell=True, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
+        print(cmd)
+        while True:
+            try:
+                line = popen.stdout.readline()
+                print("Line: ", line)
+                if not line:
+                    #None
+                    break
+           
+                try:                
+                    None
```

Ruff

Found useless expression. Either assign it to a variable or remove it.

(B018)